### PR TITLE
Feat: not using insecure & autopublish by default

### PR DESCRIPTION
**Pull request by Grubba27**
Standardized the use of insecure packages in skeletons and made this an opt-in feature. 

It is only possible to pass in this flag when you are not using the package flag( it does have a validation)

### CODE REVIEW

File: tools/cli/commands.js [hunk 1]
```diff
@@ -522,7 +522,7 @@ export const AVAILABLE_SKELETONS = [
 
 main.registerCommand({
   name: 'create',
-  maxArgs: 1,
+  maxArgs: 2,
```
Comment (denihs) [inline tools/cli/commands.js:525]: Are you sure this number needs to be increased?

**Review by denihs — CHANGES_REQUESTED** [review]
Everything looks good, but we have some tests failing. You will probably have to start passing this new flag to the apps created in the tests.

File: tools/cli/commands.js (hunk 1 above)
Comment (Grubba27) [reply tools/cli/commands.js]: https://github.com/meteor/meteor/blob/4f16220fa648660ed24fde36f99942e89d7f6c8a/tools/cli/main.js#L163
yeah, I miss understood what I read :( We can have as many options as we want but obligatory args is just one (the path)

File: tools/cli/commands.js [hunk 2]
```diff
@@ -547,7 +548,13 @@ main.registerCommand({
   // latest release to create a package if we are inside an app)
   if (options.package) {
     var packageName = options.args[0];
-
+    if (options.prototype) {
+      Console.error(
+        `The ${Console.command('--prototype')} option is not supported for packages.`
```
Comment (StorytellerCZ) [inline tools/cli/commands.js:553]: Might be better to say `is no longer supported` and add a link to some article explaining why.

File: tools/cli/help.txt [hunk 3]
```diff
@@ -191,6 +191,7 @@ Options:
   --tailwind    Create a basic react-based app, with tailwind configured.
   --chakra-ui   Create a basic react-based app, with chakra-ui configured.
   --solid       Create a basic solid-based app.
+  --prototype   Create a prototype app.
```
Comment (StorytellerCZ) [inline tools/cli/help.txt:194]: More explanation is needed here. This is too plain of a description.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: tools/cli/help.txt (hunk 3 above)
Comment (Grubba27) [reply tools/cli/help.txt]: I've rewrote like this:
``Create a prototype app with the insecure & autopublish packages. Can be used along with other app commands`` what do you think?

Comment (StorytellerCZ) [reply tools/cli/help.txt]: Maybe expand that they are for prototyping, but this is good as well.

**Review by henriquealbert — APPROVED** [review]
lgtm, just make sure to update all tutorials and everything that mention and uses autopublish and insecure @Grubba27

**Comment by retani** [comment]
Being a long term user and promoter of meteorjs I am just super happy about this. Insecure created so much confusion. Very grateful. Thanks all involved. 🙏🏼

**Pull request merged**